docs/governance/templates/release_issue.md

```diff
@@ -32,27 +32,32 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Any issues in the current milestone that are not closed, move to next milestone.
 - [ ] `git checkout main && git pull --rebase upstream main`
 - [ ] Run `make release-deploy-site`
-- [ ] Run `make build-release` to generate the CHANGELOG.md and to ensure all example images exist on us-docker.pkg.dev/agones-images/examples.
-- [ ] Download all the artifacts from the cloud build.
-- [ ] Move the CHANGELOG.md to the root of this repository, replacing any previous versions.
-- [ ] Ensure the [helm `tag` value][values] is correct (should be {version})
-- [ ] Ensure the [helm `Chart` version values][chart] are correct (should be {version})
+      -For example, if you are creating the {version} release, then you would deploy the {version}-1 service (release minus one, and then replace dots with dashes) [check for this change in GCP -> App Engine/Service]
```

Comment:
```suggestion
      - For example, if you are creating the {version} release, then this would deploy the {version}-1 service (release minus one, and then replace dots with dashes) [check for this change in GCP -> App Engine/Service]
```

Also was the part in `(release minus one, and then replace dots with dashes) [check for this change in GCP -> App Engine/Service]` meant to be a markdown link?

Reply:
It isn't a link, just to confirm the successful deployment in the app engine/ service. Added almost all the steps from the release video here😀